pkg/storage/storage.go

```diff
@@ -56,8 +56,7 @@ func RelationshipTupleReaderFromContext(ctx context.Context) (RelationshipTupleR
 // PaginationOptions should not be instantiated directly. Use NewPaginationOptions.
 type PaginationOptions struct {
 	PageSize int
-	// From is a continuation token that can be used to retrieve the next page of results. It may be also overloaded
-	// to represent a starting time for the first page of results.
+	// From is an ulid that can be used to retrieve the next page of results
 	From string
```

Comment:
I do wonder where it would make sense to rename `From` to `Ulid` or `NextUlid`. @openfga/backend any thoughts? It could come bite us in the future if for some other API, `from` doesn't equate to a ULID.

Reply:
Let's postpone this for now, as `from` isn't always a `ulid`.